lib/middlewares/csp.js

```diff
@@ -38,7 +38,7 @@ module.exports = options => {
       value = matchedOption[key];
       value = Array.isArray(value) ? value : [ value ];
 
-      // sandbox 单独处理，其他数组拆开为字符串
+      // Other arrays are splitted into strings EXCEPT `sandbox`
```

Comment:
are splitted into -> are split into

Reply:
Why? `split` is a verb, this should use a passive voice.

Reply:
Because `split` 's passive voice is `split`, not `splitted`